lib/bn.js

```diff
@@ -52,6 +52,7 @@ else
 
 BN.BN = BN;
 BN.wordSize = 26;
+BN._hasBuffer = typeof Buffer !== 'undefined';
```

Comment:
Good catch, just checked and it does include the browserify version of Buffer.

However ignoring that module alleviates it: ```browserify --ignore buffer lib/bn.js -s BN -o bn-browser.js```